### PR TITLE
refactor(sass): improve all components (S01E06) - INNO-1020

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

form-select, form-text-input, form-textarea, gallery-base, inpage-nav, label, lang-select-site, lang-select-page

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 20e882e50ec4a5664c6293dcb9652ce90a922745

https://deploy-preview-652--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 86dc4d3d09cac19d13c1dd2e9396502d064743c7

https://deploy-preview-652--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 9ae5c1a3f161e9fd8643f40f35e544c22e40879c

https://deploy-preview-652--europa-component-library.netlify.com

**Review by yhuard — DISMISSED** [review]
Please remove gallery

**Comment by emeryro** [comment]
Modifications on gallery have been reverted

File: src/generic/generic-component-form-select/generic-component-form-select.scss [hunk 1]
```diff
@@ -3,67 +3,92 @@
...
-  font-family: $ecl-font-family-sans-serif;
-  font-size: map-get($ecl-font-size, 's');
-  margin: 0;
-  padding: map-get($ecl-spacing, 'xxs')
-    ($ecl-select-background-width + map-get($ecl-spacing, 'xxxs'))
-    map-get($ecl-spacing, 'xxs') map-get($ecl-spacing, 'xxs');
-  width: 100%;
+@mixin ecl-select() {
+  .ecl-select {
+    @include ecl-select-list-arrow($ecl-color-shade);
```
Comment (yhuard) [inline src/generic/generic-component-form-select/generic-component-form-select.scss]: `ecl-select-list-arrow` is used before it's defined, this is not a good practice (readability)

**Review by yhuard — CHANGES_REQUESTED** [review]

**Review by yhuard — APPROVED** [review]

**Review by yhuard — APPROVED** [review]

**Pull request merged**